src/test/scala/com/danielasfregola/twitter4s/http/clients/rest/media/TwitterMediaClientSpec.scala

```diff
@@ -1,42 +1,167 @@
-package com.danielasfregola.twitter4s.http.clients.rest.media
```

Comment:
This file needs to be reverted

Reply:
which one, On which i have done a commit recently ??

Reply:
You PR is trying to also change the file com/danielasfregola/twitter4s/http/clients/rest/media/TwitterMediaClientSpec.scala ... which is wrong :)

Reply:
TwitterMediaClientSpec.scala  m unable to find this file please help